### ISSUE TITLE
[BUG] Invalid PIP-526 type comments make Pycln crashes with an UnparsableFile error

### ISSUE THREAD

**Issue by jensens** [hadialqattan/pycln#58]
Error on comment line.

**To Reproduce** Steps to reproduce the behavior:

1. Take this code snippet:

    ```python
    def addPortalMessage(message, type='info', request=None):
        # Note: no docstring please, to avoid reflected XSS.
        # This might not be possible, but type="structure" below sounds dangerous,
        # although I find no support for it in code.

        # The arguments are:
        #     message:   a string, with the text message you want to show,
        #                or a HTML fragment (see type='structure' below)
        #     type:      optional, defaults to 'info'. The type determines how
        #                the message will be rendered, as it is used to select
        #                the CSS class for the message. Predefined types are:
        #                'info' - for informational messages
        #                'warning' - for warning messages
        #                'error' - for messages about restricted access or
        #                          errors.

        pass


    ```

    Code reduced from https://github.com/plone/Products.CMFPlone/blob/58352686898521dd9da24d0239fd9848d04a02c5/Products/CMFPlone/PloneTool.py#L538-L567

2. Run this Pycln command:
   ```bash
   $ pycln testfile.py
   ```
3. Error traceback or unexpected output (if present):

   ```bash
   pyctnfailure.py .:1:1 SyntaxError: invalid syntax "optional, defaults to 'info'. The type determines how" ⛔

   ```

4. Unexpected fixed code (if present):

    Does not apply

**Expected behavior**:

1.  Description: A clear and concise description of what you expected to happen.

    No error, just pass and let file as is.

2.  Expected output (if present):

    No error.

3.  Expected fixed code (if present):

    Does not apply

**Environment (please complete the following informations):**

- Python Version: 3.9.5
- Pycln Version: 0.0.3
- OS Type: Linux

**Comment by hadialqattan** [comment]
Hi @jensens,
⠀
the real problem is `Pycln` crashes when it meets invalid `PIP-526` type comments as shown in your code (I know you're not using `PIP-526` but the AST parser is considering any `type: ...` as a `PIP-526` node),
⠀
thanks for the feedback.

---
PEP 526 -- Syntax for Variable Annotations:
- https://www.python.org/dev/peps/pep-0526/
- https://docs.python.org/3.8/library/ast.html#ast.parse

**Comment by hadialqattan** [comment]
This bug has been fixed #59, the fix is available at the `master` branch and it would be added in the next release (`0.0.4`).
